pysat/instruments/pysat_testing.py

```diff
@@ -85,24 +92,32 @@ def init(self):
     self.new_thing=True        
                 
```

Comment:
Alternative method for changing file dates that doesn't require overloading tag field. 

here in init()
if 'file_date_range' in self.kwargs:
    # set list files routine to desired date range
    # attach to the instrument object
    self._list_rtn = functools.partial(list_files, 
                                         file_date_range=self.kwargs['file_date_range'])
    self.files.refresh()

and modify the list_files to be
def list_files(tag=None, sat_id=None, data_path=None, format_str=None,
                     file_date_range=None):
    etc.

Reply:
It stripped of all the spaces in the code.

Reply:
I like this solution better than what I did.  I changes things, using this as an example.